### ISSUE TITLE
Error executing maven package

### ISSUE THREAD

**Issue by william-vw** [SANSA-Stack/SANSA-Template-Maven-Spark#3]
Hi everyone,

I'm getting the following error when running "mvn clean package" (it seems to be different from the previous maven-related issue):

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 10.024 s
[INFO] Finished at: 2019-06-01T15:53:19-04:00
[INFO] Final Memory: 22M/217M
[INFO] ------------------------------------------------------------------------
ERROR] Failed to execute goal on project SANSA-Template-Maven-Spark: Could not
resolve dependencies for project net.sansa-stack:SANSA-Template-Maven-Spark:jar:
0.5.1-SNAPSHOT: Failed to collect dependencies at net.sansa-stack:sansa-rdf-spar
k_2.11:jar:0.5.1-SNAPSHOT: Failed to read artifact descriptor for net.sansa-stac
k:sansa-rdf-spark_2.11:jar:0.5.1-SNAPSHOT: Could not transfer artifact net.sansa
-stack:sansa-rdf-spark_2.11:pom:0.5.1-SNAPSHOT from/to maven.aksw.snapshots (htt
p://maven.aksw.org/archiva/repository/snapshots): sun.security.validator.Validat
orException: PKIX path building failed: sun.security.provider.certpath.SunCertPa
thBuilderException: unable to find valid certification path to requested target
```

I'm eager to try out Sansa so it'd be great if I could resolve this maven issue! :-)

**Comment by GezimSejdiu** [comment]
Hi @william-vw ,

thanks a lot for the issue. In order to test it (even though my .m2/ folder wasn't empty) I did clone it and try `mvn -U install` and I get :
```bash
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:39 min
[INFO] Finished at: 2019-06-06T11:05:20+02:00
[INFO] ------------------------------------------------------------------------
```
After that, I did `mvn clean package` and the result is as following: 
```sh
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:07 min
[INFO] Finished at: 2019-06-06T11:07:10+02:00
[INFO] ------------------------------------------------------------------------
```
Could you please, do `mvn -U install` first and see if that resolve the issue? If not then we have to double check it what is causing this security issue at your end.

Best regards,

**Comment by william-vw** [comment]
Hi @GezimSejdiu,

Regrettably it still fails with the same validator exception..

**Comment by furkanozbay** [comment]
Hi, I have same error too.

[ERROR] Failed to execute goal on project SANSA-Template-Maven-Spark: Could not resolve dependencies for project net.sansa-stack:SANSA-Template-Maven-Spark:jar:0.6.0: The following artifacts could not be resolved: com.ibm.sparktc.sparkbench:sparkbench:jar:2.3.0_0.4.0, org.netbeans.modules:org-netbeans-modules-masterfs:jar:RELEASE82, org.netbeans.modules:org-netbeans-core-startup-base:jar:RELEASE82, org.netbeans.modules:org-netbeans-core:jar:RELEASE82, org.netbeans.api:org-openide-util-lookup:jar:RELEASE82, org.netbeans.api:org-openide-util:jar:RELEASE82, org.netbeans.api:org-openide-filesystems:jar:RELEASE82, org.netbeans.api:org-openide-util-ui:jar:RELEASE82, org.netbeans.api:org-openide-dialogs:jar:RELEASE82, org.netbeans.api:org-openide-nodes:jar:RELEASE82, org.netbeans.api:org-netbeans-api-annotations-common:jar:RELEASE82, net.java.dev:stax-utils:jar:snapshot-20100402, com.github.jeremyh:jBCrypt:jar:jbcrypt-0.4: Could not find artifact com.ibm.sparktc.sparkbench:sparkbench:jar:2.3.0_0.4.0

**Comment by LorenzBuehmann** [comment]
Has been fixed in version 0.6.1-SNAPSHOT by SANSA-RDF layer commit https://github.com/SANSA-Stack/SANSA-RDF/commit/fbcb080bd1b5bb93bac3a9e60519a0c741f19f61
